src/cljs/rems/application.cljs

```diff
@@ -602,23 +595,32 @@
                              :readonly readonly?
                              :app-id (:application/id application))]))]}]))
 
-(defn- application-licenses [application edit-application]
+(defn- application-licenses [application edit-application userid]
   (when-let [licenses (not-empty (:application/licenses application))]
-    (let [accepted-licenses (:accepted-licenses edit-application)
+    (let [application-id (:application/id application)
+          accepted-licenses (get (:application/accepted-licenses application) userid)
+          possible-commands (:application/permissions application)
           license-validations (index-by [:license-id] (:validation-errors edit-application))
           form-fields-editable? (form-fields-editable? application)
           readonly? (not form-fields-editable?)]
       [collapsible/component
        {:id "form"
         :title (text :t.form/licenses)
         :always
-        [:div.form-group.field
+        [:div
+         [:p (text :t.form/must-accept-licenses)]
```

Comment:
should this be shown only when licenses haven't yet been accepted?

Reply:
Now the wording is more neutral